dashboard/app/controllers/ai_tutor_interactions_controller.rb

```diff
@@ -0,0 +1,61 @@
+class AiTutorInteractionsController < ApplicationController
+  include Rails.application.routes.url_helpers
+  before_action :authenticate_user!
+  load_and_authorize_resource :ai_tutor_interaction
+
+  # POST /ai_tutor_interactions
```

Comment:
oh! Are you thinking we'd move all of the `ai_tutor_interactions_controller` logic into the `openai_chat_controller`? I'm a little bit leery of that because it feels a little more tangled up in a specific model (e.g. OpenAI) and a little less reusable, but maybe I'm misunderstanding the suggestion....

Reply:
I've thought about it more, and don't block yourself on this feedback! We can change later if we want one less API call.